java/ql/src/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -391,10 +390,8 @@ private predicate taintPreservingQualifierToMethod(Method m) {
     )
   )
   or
-  m instanceof StringFormatMethod
-  or
   m.getDeclaringType() instanceof TypeFormatter and
-  m.hasName("out")
+  m.hasName(["format", "out"])
```

Comment:
Doesn't `String::formatted` also preserve taint from the qualifier to the result?

Reply:
`formatted` is in fact already modelled in the same predicate